x-pack/filebeat/module/ibmmq/errorlog/_meta/fields.yml

```diff
@@ -0,0 +1,42 @@
+- name: errorlog
+  description: IBM MQ error logs
+  example: Please add example
+  type: group
+  fields:
+  - name: installation
+    description: Please add description
+    example: Please add example
+    type: keyword
+  - name: qmgr
```

Comment:
@felix-lessoer Can I get some details on what qmgr is?

Reply:
This is the name of the queue manager where the error has occurred on. 

From IBM documentation
Queue managers provide queuing services to applications, and manages the queues that belong to them. A queue manager ensures the following actions:
Object attributes are changed according to the commands received.
Special events such as trigger events or instrumentation events are generated when the appropriate conditions are met.
Messages are put on the correct queue, as requested by the application making the MQPUT call. The application is informed if this cannot be done, and an appropriate reason code is given.

Reply:
updated.